src/blob.rs

```diff
@@ -312,6 +313,27 @@ impl<'a> BlobObject<'a> {
         true
     }
 
+    /// Returns path to the stored Base64-decoded blob.
+    pub(crate) async fn store_from_base64(
```

Comment:
if this expects an image then it makes sense to have `image` in the function name.

otherwise saving blobs from base64 would also be interesting in general, like allowing to send blobs to core via jsonrpc when you either don't have access to the real file paths (that will likely be the case on KaiOS) or for a DC web where you would use DC in a browser and the core runs on your phone.

Reply:
Currently it's used only for images, but it can be used for anything. It just tries to guess a file extension if an image is actually passed. So, i'm unsure we need `image` in the function name, but anyway better ideas are welcome

Reply:
> So, i'm unsure we need image in the function name

I think at least in the doc comment as long as it is specialised for images.

Reply:
Appended the doc comment